packages/core/data-transfer/src/strapi/providers/local-source/assets.ts

```diff
@@ -61,6 +61,30 @@ function getFileStats(filepath: string, isLocal = false): Promise<{ size: number
       });
   });
 }
+
+async function signUrl(isLocalProvider: boolean, file: any) {
+  if (isLocalProvider) return;
+  const { provider } = strapi.plugins.upload;
+  const { provider: providerName } = strapi.config.get('plugin.upload') as any;
+  const isPrivate = await provider.isPrivate();
+  if (file.provider === providerName && isPrivate) {
+    const signUrl = async (file: any) => {
```

Comment:
It's a bit confusing to have this function being named the same as the parent one

Reply:
Also, could we use unknown instead of any when possible?